index.html

```diff
@@ -815,95 +912,128 @@ <h3>
 }
       </pre>
 
-      <p>
-        If this DID document is consumed as `application/did+json` by an implementation that doesn't understand JSON-LD,
-        it will interpret `blockchainAccountId` as the <a
-              href="https://w3c.github.io/DID-SPEC-REGISTRIES/#blockchainaccountid">property listed in the DID
-        Specification Registries</a>, and it will process it accordingly.
-      </p>
+        <p>
+          If this DID document is consumed as `application/did+json` by an
+          implementation that doesn't understand
+          JSON-LD,
+          it will interpret `blockchainAccountId` as the <a
+            href="https://w3c.github.io/did-spec-registries/#blockchainaccountid">property
+            listed in the DID
```

Comment:
This should be changed to `https://www.w3.org/TR/did-spec-registries/#blockchainaccountid` (or its `data-cite` equivalent). It is not appropriate to refer to the editor's draft if there is an official /TR document.